### PR TITLE
Feat/demo mpc mod

### PR DESCRIPTION
**Pull request by arch1995**

### CODE REVIEW

File: demo/tss-vue-ts-app/src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      upper = upper.umod(ec.curve.n);
+      let tempLower = myIndex.sub(allIndexes[j]);
+      tempLower = tempLower.umod(ec.curve.n);
+      lower = lower.mul(tempLower).umod(ec.curve.n);
+    }
+  }
+  return upper.mul(lower.invm(ec.curve.n)).umod(ec.curve.n);
+}
+
+export function getTSSPubKey(dkgPubKey: PointHex, userSharePubKey: PointHex, userTSSIndex: number): any {
```
Comment (himanshuchawla009) [inline demo/tss-vue-ts-app/src/utils.ts:92]: get tss pub key directly from tkey sdk, no need of this function with tkey.

**Review by himanshuchawla009 — CHANGES_REQUESTED** [review]

File: demo/tss-vue-ts-app/src/App.vue [hunk 2]
```diff
@@ -217,55 +165,127 @@ export default Vue.extend({
...
+
+    await this.torusDirectSdk.init({ skipSw: false });
+
+    const PRIVATE_KEY = generatePrivate().toString("hex");
+
+    this.torusSp = new TorusServiceProvider({
+      postboxKey: PRIVATE_KEY,
+      useTSS: true,
+      customAuthArgs: {
+        baseUrl: "http://localhost:3000",
```
Comment (himanshuchawla009) [inline demo/tss-vue-ts-app/src/App.vue:246]: will it work after hosting?

File: demo/tss-vue-ts-app/src/App.vue [hunk 3]
```diff
@@ -217,55 +165,127 @@ export default Vue.extend({
...
+      },
+    });
+
+    const torusSL = new MockStorageLayer();
```
Comment (himanshuchawla009) [inline demo/tss-vue-ts-app/src/App.vue:250]: we should use metadata server other wise, account wont persist after page reload

**Review by himanshuchawla009 — CHANGES_REQUESTED** [review]

File: demo/tss-vue-ts-app/src/App.vue (hunk 3 above)
Comment (arch1995) [reply demo/tss-vue-ts-app/src/App.vue:250]: hey, I am saving the device share in local storage, after I refresh and re-login, the account hash is same

**Pull request merged**